\providecommand{\main}{..} 
\documentclass[\main/boa.tex]{subfiles}

\begin{document}

\section{RNA-seq transcriptional profiling of PPD-b-stimulated peripheral blood
from cattle infected with \emph{Mycobacterium bovis}}

\begin{center}
  {\bf \index[a]{Correia Carolina N.} Carolina N. Correia$^{1^\star}$, \index[a]{McLoughlin Kirsten E.} Kirsten E. McLoughlin$^{1}$, \index[a]{Nalpas Nicolas C.} Nicolas C. Nalpas$^{1}$, \index[a]{Magee David A.} David A. Magee$^{1}$, \index[a]{Browne John A.} John A. Browne$^{1}$, \index[a]{Killick Kate E.} Kate E. Killick$^{1}$, \index[a]{Vordermeier H. Martin} H. Martin Vordermeier$^{2}$, \index[a]{Villarreal-Ramos Bernardo } v$^{2}$, \index[a]{Gordon Stephen V.} Stephen V. Gordon$^{3}$, \index[a]{MacHugh David E.} David E. MacHugh$^{1}$}
\end{center}

\vskip 0.3cm

\begin{affiliations}
\begin{enumerate}
\begin{minipage}{0.915\textwidth}
\centering
\item Animal Genomics Laboratory, UCD School of Agriculture and Food Science,
University College Dublin \\[-2pt]
\item Animal and Plant Health Agency, UK \\[-2pt]
\item UCD School of Veterinary Medicine, University College Dublin \\[-2pt]
\end{minipage}
\end{enumerate}
$^\star$Contact author: \href{mailto:carolina.correia@ucdconnect.ie}{\nolinkurl{carolina.correia@ucdconnect.ie}}\\
\end{affiliations}

\vskip 0.5cm

\begin{minipage}{0.915\textwidth}
\keywords transcriptomics; differential gene expression; tuberculosis
\packages \index[p]{AnnotationFuncs} AnnotationFuncs; \index[p]{magrittr} magrittr; \index[p]{VennDiagram} VennDiagram; \index[p]{MASS} MASS; \index[p]{RColorBrewer} RColorBrewer; \index[p]{svglite} svglite; \index[p]{org.Bt.eg.db} org.Bt.eg.db; \index[p]{dplyr} dplyr; \index[p]{edgeR} edgeR; \index[p]{limma} limma; \index[p]{extrafont} extrafont; \index[p]{ggplot2} ggplot2; \index[p]{gridExtra} gridExtra
\end{minipage}

\vskip 0.8cm

\emph{Mycobacterium bovis} infection, the cause of bovine tuberculosis
(BTB), costs an estimated \$3 billion to global agriculture annually.
During the last decade, the maturation of high-throughput sequencing
technologies coupled with well-annotated genome resources, has provided
an unprecedented opportunity to gain a deeper understanding of
host-pathogen interactions for many infectious diseases. Within this
context, transcriptional profiling of the host immune response to
\emph{M. bovis} infection is a powerful approach for identifying host
genes and cellular pathways important to disease pathology. For the
present study, ten age-matched male Holstein-Friesian calves were
infected endobronchially with \emph{M. bovis} (\textasciitilde{}2,000
CFU -- colony forming units). Peripheral blood samples were collected in
duplicate at four time points (-1 week pre-infection, +1 week, +2 week,
and +10 week post-infection) and used for:

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  an overnight stimulation with purified protein derivative of bovine
  tuberculin (PPD-b) at 37\(^\circ\)C
\item
  a control overnight incubation at 37\(^\circ\)C without PPD-b
  stimulation.
\end{enumerate}

After isolation of total RNA, poly(A)+ purified RNA was used to generate
strand-specific RNA-seq libraries for high-throughput sequencing.
Transcripts were quality checked, adapter and quality filtered, and then
aligned to the \emph{Bos taurus} reference genome UMD3.1.1. Following
summarisation of gene counts, lowly expressed transcripts were removed
prior to subsequent gene annotation and differential expression
analyses. Results showed 929 differentially expressed (DE) genes at -1
week pre-infection, 1,619 DE genes +1 week post-infection, 1,170 DE
genes at +2 week, and 5,535 DE genes at +10 week (compared to
non-PPDb-stimulated group at each time point; FDR correction threshold
\(\leq\) 0.05).

\end{document}
